spec/controllers/queue_items_controller_spec.rb

```diff
@@ -78,101 +82,158 @@
       let(:action) { delete :destroy, id: 1 }
     end
   end
-  
+
   describe "POST update_queue" do
     it_behaves_like "requires sign in" do
-      let(:action) { post :update_queue, queue_items: [{id: 1, position: 2}, {id: 2, position: 1}] }
+      let(:action) do
+        post :update_queue, queue_items: [
+          { id: 1, position: 2 },
+          { id: 2, position: 1 }
+        ]
+      end
     end
+
     context "with valid inputs" do
       let(:user1) { Fabricate(:user) }
       let(:video) { Fabricate(:video) }
-      let(:queue_item1) { Fabricate(:queue_item, user: user1, position: 1, video: video) }
-      let(:queue_item2) { Fabricate(:queue_item, user: user1, position: 2, video: video) }
-    
+      let(:queue_item1) { Fabricate(:queue_item,
+                                    user: user1,
+                                    position: 1,
+                                    video: video) }
+      let(:queue_item2) { Fabricate(:queue_item,
+                                    user: user1,
+                                    position: 2,
+                                    video: video) }
+
       before { set_current_user(user1) }
-      
+
       it "redirects to the my queue page" do
-        post :update_queue, queue_items: [{id: queue_item1.id, position: 2}, {id: queue_item2.id, position: 1}]
+        post :update_queue,
+          queue_items: [
+            { id: queue_item1.id, position: 2 },
+            { id: queue_item2.id, position: 1 }
+          ]
         expect(response).to redirect_to my_queue_path
       end
-      
+
       it "reorders the queue items" do
-        post :update_queue, queue_items: [{id: queue_item1.id, position: 2}, {id: queue_item2.id, position: 1}]
+        post :update_queue,
+          queue_items: [
+            { id: queue_item1.id, position: 2 },
+            { id: queue_item2.id, position: 1 }
+           ]
         expect(user1.queue_items).to eq([queue_item2, queue_item1])
       end
-      
+
       it "normalizes the position numbers" do
-        post :update_queue, queue_items: [{id: queue_item1.id, position: 3}, {id: queue_item2.id, position: 4}]
+        post :update_queue,
+          queue_items: [
+            { id: queue_item1.id, position: 3 },
+            { id: queue_item2.id, position: 4 }
+          ]
         expect(user1.queue_items.map(&:position)).to eq([1,2])
       end
     end
-    
+
     context "with invalid inputs" do
       let(:user1) { Fabricate(:user) }
       let(:video) { Fabricate(:video) }
-      let(:queue_item1) { Fabricate(:queue_item, user: user1, position: 1, video: video) }
-      let(:queue_item2) { Fabricate(:queue_item, user: user1, position: 2, video: video) }
-    
+      let(:queue_item1) { Fabricate(:queue_item,
+                                    user: user1,
+                                    position: 1,
+                                    video: video) }
+      let(:queue_item2) { Fabricate(:queue_item,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.